packages/paste-core/components/chat-log/__tests__/ChatBubble.spec.tsx

```diff
@@ -26,7 +27,7 @@ const CustomizationFooWrapper: React.FC = ({children}) => (
     baseTheme="default"
     theme={TestTheme}
     elements={{
-      foo_bubble: {
+      FOO_BUBBLE: {
```

Comment:
question: were these changed for the `camelCase` rule?

Reply:
Yup!